package/MDAnalysis/analysis/contacts.py

```diff
@@ -368,11 +368,18 @@ class Contacts(AnalysisBase):
     Attributes
     ----------
     timeseries : list
-        list containing *Q* for all refgroup pairs and analyzed frames
-
+        list containing *Q* for all refgroup pairs and analyzed frames 
+    r0 : list
+        list containing the initial distance array in the reference structure
+    initial_contacts : list
+        list containing the contact_matrix based on the radius cutoff 
```

Comment:
I'm a bit rusty on the `contacts` code, but I think here you should be saying here that it's the contact matrix based on the reference frame.